### PR TITLE
Adding new sandbox projects!

### PR DESCRIPTION
**Pull request by amye**
Adding OpenFunction, Teller and Sealer

### CODE REVIEW

File: docs/projects.md [hunk 1]
```diff
@@ -167,7 +170,10 @@
 | [LitmusChaos](https://litmuschaos.io)   |  6/23/20        | 1/11/22           |[Incubating](https://github.com/cncf/toc/pull/569)|
 | [OpenMetrics](https://github.com/OpenObservability/OpenMetrics)|8/10/18| 2/2/2022 | [Incubating](https://github.com/cncf/toc/pull/615)|
 | [ChaosMesh](https://github.com/pingcap/chaos-mesh)|7/14/20| 2/16/2022 | [Incubating](https://github.com/cncf/toc/pull/683) |
-| [KubeVirt](https://kubevirt.io)         |  9/9/19        | 4/19/22           |[Incubating](https://github.com/cncf/toc/pull/715)|
```
Comment (jeefy) [inline docs/projects.md:170]: Did you mean to move Kubevirt into Graduating with this PR? :thinking:

**Review by jeefy — CHANGES_REQUESTED** [review]
LGTM besides one minor question

**Comment by amye** [comment]
LOL no, KubeVirt was already added in here, this was catchup on the ones that I am behind on.
This is why we review!

File: docs/projects.md (hunk 1 above)
Comment (amye) [reply docs/projects.md:170]: ## Sandbox to Incubation Project Moves - line 149. The headers just didn't collapse properly in GH.

**Comment by amye** [comment]
> LOL no, KubeVirt was already added in here, this was catchup on the ones that I am behind on. This is why we review!

Now that I've had a chance to review, GH just smashed the headers together, it's in the correct place.

**Pull request merged**